lib/metrics.ts

```diff
@@ -51,49 +32,34 @@ function prepareData(res: LighthouseResults): MetricsResults {
   const colorVisual = 'blue';
 
   const timings: Timing[] = [];
-  const navStart = metricsDefinitions.find(def => def.id === metricsIds.NAVSTART);
-  const timestamps: Timestamp[] = [{
-    title: navStart.name,
-    id: navStart.id,
-    timestamp: navStart.getTs(audits)
-  }];
 
-  metricsDefinitions
-    // exclude navStart as its not a timing
-    .filter(def => def.id !== metricsIds.NAVSTART)
-    // exclude experimental metrics
-    .filter(def => def.id !== metricsIds.TTI
-      && def.id !== metricsIds.OnLoad
-      && def.id !== metricsIds.EndOfATrace)
-    .forEach(metric => {
-      const resolvedMetric: Timing = {
-        title: metric.name,
-        id: metric.id,
-        timestamp: metric.getTs(audits),
-        timing: (metric.getTs(audits) - navStart.getTs(audits)) / 1000,
-        color: colorVisual
-      };
+  Object.keys(audits).forEach(metricKey => {
+    const metric = audits[metricKey];
+    const resolvedMetric: Timing = {
+      title: metric.title,
+      id: metric.id,
+      timing: typeof metric.rawValue === 'boolean' ? 0 : metric.rawValue,
+      color: colorVisual
+    };
 
-      switch (metric.id) {
-        case metricsIds.TTFCP:
-        case metricsIds.TTFMP:
-          resolvedMetric.color = colorP2;
-          break;
-        case metricsIds.TTFI:
-        case metricsIds.TTCI:
-          resolvedMetric.color = colorP0;
-          break;
-      }
+    switch (metric.id) {
+      case metricsIds.TTFMP:
+        resolvedMetric.color = colorP2;
+        break;
+      case metricsIds.TTFI:
+      case metricsIds.TTCI:
+        resolvedMetric.color = colorP0;
+        break;
+    }
 
-      timings.push(resolvedMetric);
-    });
+    timings.push(resolvedMetric);
+  });
 
   return {
     timings,
-    timestamps,
-    generatedTime: res.generatedTime,
+    generatedTime: res.fetchTime,
     lighthouseVersion: res.lighthouseVersion,
-    initialUrl: res.initialUrl,
-    url: res.url
+    initialUrl: res.finalUrl,
```

Comment:
initial is final? 

i'd recommend adopting `requestedUrl` and `finalUrl` names for this metrics export.